Add some Inventories and Terraform templates (#276)
OpenSRP/playbooks: README.md

    └ ...
```

You could alternatively use the included script `scripts/new_inventory.sh` to generate from the sample inventories a specific application. For example:

```console
./scripts/new_inventory.sh reveal_web demo production
```

Generates the following inventory files:

```console
inventories
├── demo

> │   └── staging

Reviewer: Should this be `production` to match the command above?

> │   └── staging

Reviewer 2: ```suggestion
│   └── production
```

│       └── group_vars
│           ├── reveal_web
│           │   ├── vars.yml
│           │   └── vault.yml
│           └── reveal_web_client
│               └── vars.yml
```

>    **Note**: The hosts file is not part of the file generated. You will need to generate ypurself, look at the `sample-inventories/inventory-a/hosts` for an example.

Reviewer 3: ```suggestion
   **Note**: The hosts file is not part of the file generated. You will need to generate yourself, look at the `sample-inventories/inventory-a/hosts` for an example.
```

>    **Note**: The hosts file is not part of the file generated. You will need to generate ypurself, look at the `sample-inventories/inventory-a/hosts` for an example.

Reviewer: ```suggestion
   **Note**: The hosts file is not part of the file generated. You will need to generate it yourself, look at the `sample-inventories/inventory-a/hosts` for an example.
```
